src/components/Testing.tsx

```diff
@@ -16,27 +48,66 @@ export default function Testing(user : Struct.User) {
   // function test (){
   //     console.log('this works yay!!');
   //     // number = (number + 10 ) %20;
+
+
   function LogEvent(){
     console.log(count)
     console.log(dummyEvent[count]);
     console.log(Struct.timeCalc(dummyEvent[count]));
 
   }
-  // }
+  
+  async function testo(){
+    // const testl = new API();
+    // console.log( await testl.getEvent(1));
+    let dd : Struct.Event = {
+    id: 17,
```

Comment:
where can I see this testing event?

Reply:
You will need to do checks in Supabase to see the data. Alternatively, you can test it with the other functions. For example, create (x), then call read(x) etc.